cfme/containers/node.py

```diff
@@ -81,17 +84,15 @@ def is_displayed(self):
     paginator = PaginationPane()
 
 
-class Node(Taggable, Labelable, SummaryMixin, Navigatable):
+class Node(Taggable, Labelable, SummaryMixin, NavigatableMixin):
 
     PLURAL = 'Nodes'
 
     def __init__(self, name, provider, collection=None, appliance=None):
```

Comment:
I think we can remove ```appliance=None``` argument right? since we get this from the collection.
or maybe we can do instead:
```python
self.appliance = self.collection.appliance or appliance
```